src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -116,48 +113,43 @@ const NavMenu = (): ReactElement => {
                         id="menu-list-grow"
                       >
                         <NextLink
-                          href="/accountLists/[accountListId]/reports/donations"
-                          as={`/accountLists/${state.accountListId}/reports/donations`}
+                          href={`/accountLists/${state.accountListId}/reports/donations`}
```

Comment:
This should use your new hook someday. Eventually we should make a cleanup PR to get rid of `useApp` but that can happen later.